TOC.md

```diff
@@ -92,13 +92,149 @@
       - [Identify Slow Queries](/identify-slow-queries.md)
       - [Identify Expensive Queries](/identify-expensive-queries.md)
   + Scale
+<<<<<<< HEAD
```

Comment:
@toutdesuite Please resolve the conflict. Be careful about the version difference.

Reply:
@TomShawn PTAL, thanks